gc/base/standard/ParallelGlobalGC.cpp

```diff
@@ -1645,7 +1645,7 @@ MM_ParallelGlobalGC::reportGCIncrementStart(MM_EnvironmentBase *env)
 		J9HOOK_MM_PRIVATE_GC_INCREMENT_START,
 		stats);
 }
-
+// !@!@ reportGCIncrementEnd
```

Comment:
remove this comment

Reply:
I don't think PR is valid, look like debug changes were pushed to the branch